src/frontend/components/SøknadsSteg/Utvidet-DinLivssituasjon/NåværendeSamboerSkjema.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { ESvar } from '@navikt/familie-form-elements';
+import { ISkjema } from '@navikt/familie-skjema';
+
+import Datovelger from '../../Felleskomponenter/Datovelger/Datovelger';
+import KomponentGruppe from '../../Felleskomponenter/KomponentGruppe/KomponentGruppe';
+import { SkjemaCheckbox } from '../../Felleskomponenter/SkjemaCheckbox/SkjemaCheckbox';
+import { SkjemaFeltInput } from '../../Felleskomponenter/SkjemaFeltInput/SkjemaFeltInput';
+import { SamboerSpørsmålId, samboerSpørsmålSpråkId } from './spørsmål';
+import { IDinLivssituasjonFeltTyper } from './useDinLivssituasjon';
+
+const NåværendeSamboerSkjema: React.FC<{
+    skjema: ISkjema<IDinLivssituasjonFeltTyper, string>;
+}> = ({ skjema }) => {
+    return skjema.felter.harSamboerNå.verdi === ESvar.JA ? (
+        <KomponentGruppe>
```

Comment:
Hvis du legger på dynamisk-propen her så vil den få en aria-live="polite" tag som gjør at den leses opp av skjermleser i det den blir synlig for brukeren :)